src/Network/Network/ChangeLog.md

```diff
@@ -26,6 +26,7 @@
 * Updated cmdlet to add 'TunnelInterface' property for backend pool related operations.
     - `New-AzLoadBalancerBackendAddressPool`
     - `Set-AzLoadBalancerBackendAddressPool`
+* Onboard Azure Resource Manager to Private Link Common Cmdlets
```

Comment:
I think this should be move to section `Upcoming Release`.

Reply:
Opened https://github.com/Azure/azure-powershell/pull/15897 with a fix